fastlane/lib/fastlane/runner.rb

```diff
@@ -58,10 +58,14 @@ def execute(lane, platform = nil, parameters = nil)
       rescue => ex
         Dir.chdir(path_to_use) do
           # Provide error block exception without colour code
-          error_ex = ex.exception(ex.message.gsub(/\033\[\d+m/, ''))
```

Comment:
Yes, forgot to mention that in the PR body: this is not needed any more, as we now use `UI.user_error!` or `UI.crash!` instead of `"error_message".red` (which was the case in the early fastlane days